packages/intl/src/strings.ts

```diff
@@ -1675,12 +1675,15 @@ For example:
     t`Your account is now 100% secure against unauthorized logins.`,
   sms: () => t`phone number`,
   app: () => t`authentication app`,
-  mfaFallbackMethodText: (fallback: string, primary: string) =>
-    `You will now receive your 2FA codes on your ${
-      strings[fallback as keyof typeof strings]
-    } in case you lose access to your ${
-      strings[primary as keyof typeof strings]
-    }.`,
+  mfaFallbackMethodText: (
+    fallback: "app" | "sms" | "email",
+    primary: "app" | "sms" | "email"
```

Comment:
I've hardcoded the types here to appease TypeScript. I'm wondering if this is fine or should we import the `AuthenticatorType` type from the `core` package?